setup.py

```diff
@@ -38,13 +38,19 @@ def get_param(pattern):
     ],
     extras_require={
         'dev': [
+            "black==19.3b0",
             "coverage==4.5.1",
             "flake8==3.5.0",
-            "pylint==2.1.1",
+            "pylint==2.3.0",
```

Comment:
Please take the opportunity to update all the `dev` dependencies to the latest versions.

Reply:
:+1: sure!